File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
    result = ""
    
    if number % 3 == 0
        result = "Pling"
    end

    if number % 5 == 0
        result = result * "Plang"
    end

    if number % 7 == 0
        result = result * "Plong"
    end

    if result == ""
        return string(number)
    end

    result
    
end

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"
#include <string>
#include <iostream>

namespace raindrops {
std::string convert(int num) {

       std::string ns{""};

       if (num % 3 == 0) {
         ns = "Pling";
       }

       if (num % 5 == 0) {
         ns = ns + "Plang";
       }

       if (num % 7 == 0) {
         ns = ns + "Plong";
       }

       if (num % 3 != 0 && num % 5 != 0 && num %7 !=0) {
         ns=std::to_string(num);
       }

       return ns;
    }
}// namespace raindrops
